includes/class-wc-payments-token-service.php

```diff
@@ -62,7 +62,7 @@ public function add_token_to_user( $payment_method, $user ) {
 
 		if ( Payment_Method::SEPA === $payment_method['type'] ) {
 			$token = new WC_Payment_Token_WCPay_SEPA();
-			$token->set_gateway_id( CC_Payment_Gateway::GATEWAY_ID );
+			$token->set_gateway_id( 'woocommerce_payments_sepa_debit' );
```

Comment:
Nitpick: It could be worth making this a constant like we do in our JS constants file:

https://github.com/Automattic/woocommerce-payments/blob/develop/client/checkout/constants.js#L4

Reply:
```suggestion
			$token->set_gateway_id( UPE_Payment_Gateway::GATEWAY_ID . '_' . Sepa_Payment_Method::PAYMENT_METHOD_STRIPE_ID );
```
```suggestion
			$token->set_gateway_id( UPE_Payment_Gateway::GATEWAY_ID . '_' . Payment_Method::SEPA );
```
Alternatively, you could Frankenstein this together from some of the constants we already have? Otherwise creating a new `Enum` or something like that sounds like a good plan.